doc/devdocs/types.rst

```diff
@@ -319,9 +319,11 @@ the type, which is an object of type :obj:`TypeName`:
      primary: Array{T,N} <: DenseArray{T,N}
      cache: SimpleVector
        ...
+
      linearcache: SimpleVector
        ...
-     uid: Int64 47
+
+     uid: Int64 -7900426068641098781
```

Comment:
I'm just wondering why it was previously given as 47 - should check blame to see who added that, and if they might have been running on Linux. Did you run this on mac or?